### DISCUSSION TITLE
【 BUG求助 】InputNumber异常加减，附带重现代码

### DISCUSSION THREAD

**Discussion by johnsxiongs** [vueComponent/ant-design-vue#4215] [General]
复现地址：https://antd-test-nine.vercel.app/
代码仓库：https://github.com/johnxiongs/antd-test.git
<img width="936" alt="WX20210617-235041@2x" src="https://user-images.githubusercontent.com/37213061/122431255-de00ca00-cfc6-11eb-96c0-a4bcb1bd48af.png">
如图，点击输入框g，f框值会发生变化！

**Comment by johnsxiongs** [comment]
初步判定是这个描述列表引发的错误，但是具体原因还不知道，群主能帮忙看看吗？

**Comment by John60676** [comment]
`a-descriptions-item` 组件加上 `key` 即可
